pandas/core/nanops.py

```diff
@@ -353,10 +353,16 @@ def _nanvar(values, axis=None, skipna=True, ddof=1):
         values = values.copy()
         np.putmask(values, mask, 0)
 
-    X = _ensure_numeric(values.sum(axis))
-    XX = _ensure_numeric((values ** 2).sum(axis))
-    result = np.fabs((XX - X * X / count) / d)
-    return result
+    # This implementation is based on that of nanvar in numpy 1.8 and up. TODO:
+    # call directly into numpy when we bump numpy up 1.8 .
```

Comment:
put the issue number as a comment here. I would take out the reference, unless you want to add a conditional if the user has >= 1.8 and use the function, otherwise I suspect this wont' be added.